servers/rendering/renderer_rd/storage_rd/light_storage.cpp

```diff
@@ -714,7 +714,13 @@ void LightStorage::update_light_buffers(RenderDataRD *p_render_data, const Paged
 						light_data.shadow_split_offsets[j] = split;
 						float bias_scale = light_instance->shadow_transform[j].bias_scale * light_data.soft_shadow_scale;
 						light_data.shadow_bias[j] = light->param[RS::LIGHT_PARAM_SHADOW_BIAS] / 100.0 * bias_scale;
-						light_data.shadow_normal_bias[j] = light->param[RS::LIGHT_PARAM_SHADOW_NORMAL_BIAS] * light_instance->shadow_transform[j].shadow_texel_size;
+						if (j >= 1) {
```

Comment:
There is no need for the branch here since `light_data.shadow_split_offsets[0] / split` is `1.0` when `j` is `0`